lib/migrator/migrationSteps.spec.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-env jest */
+
+const path = require('path')
+const fse = require('fs-extra')
+const reporter = require('./reporter')
```

Comment:
If we add the following lines above this require, then we don't need to change the require of `./reporter` in the implementation file:

```suggestion

jest.mock('./reporter', () => {
  const mockReporter = jest.fn()
  return {
    addReporter: jest.fn().mockReturnValue(mockReporter),
    mockReporter: mockReporter,
    reportFailure: jest.fn(),
    reportSuccess: jest.fn()
  }
})

const reporter = require('./reporter')
```